src/anupSJun20/PalindromeNumber.java

```diff
@@ -0,0 +1,28 @@
+package anupSJun20;
+/*Program 1: 
+Verify given number is palindrome or not. 
+Input: number = 45654
+Output : Given number 45654 is palindrome.
+
+Input: number = 56789
+Output : Given number 56789 is not palindrome.*/
+
+public class PalindromeNumber {
+	static void findPalindromeNumber(int input) {
+		int initialNum = input;
+		int revNum = 0;
+		while (input > 0) {
+			revNum = revNum * 10 + input % 10;
```

Comment:
Here put bracket for r(evNum * 10) + (input %10) so it will be more clear understand
Logic is perfectly placed

Reply:
Added bracket and pushed again.